src/core.cpp

```diff
@@ -72,6 +81,10 @@ namespace Core
 	
 	/** The Address for the Last Round. **/
 	std::string LAST_ROUND_BLOCKFINDER = "2S4PPSznWfPVLtPJNpi8Ly46Wft3wGbayGkhaGzKLVcepmrhKTP";
+
+	unsigned int nConnections = 0;
+	unsigned int nLastBlockFound = 0;
```

Comment:
OK - was just pointing it out in case it was a symptom of a problem. Note that where it would be used, in statscollector.cpp line 339 is commented out (along with several other methods in that cpp)